### ISSUE TITLE
 Upgrading a package won't work as depending versions are not resolved properly 

### ISSUE THREAD

**Issue by NicoHaase** [chocolatey/choco#1408]
### What You Are Seeing?
Update of docker-toolbox tries to install old version of git.install which should be an "update"

### What is Expected?
Updating docker-toolbox won't downgrade a recent git installation

### How Did You Get This To Happen? (Steps to Reproduce)
Installed packages (amongst others):
docker-toolbox v17.06.0
git v 2.14.1
git.install v2.14.1
chocolatey 0.10.8
chocolatey-core.extension 1.3.1
chocolatey 0.9.9.8

Ran `cup all`

### Output Log
```
C:\WINDOWS\system32>cup --debug --verbose docker-toolbox
Chocolatey v0.10.8
Chocolatey is running on Windows v 10.0.15063.0
Attempting to delete file "C:/ProgramData/chocolatey/choco.exe.old".
Attempting to delete file "C:\ProgramData\chocolatey\choco.exe.old".
Command line: "C:\ProgramData\chocolatey\choco.exe" upgrade --debug --verbose docker-toolbox
Received arguments: upgrade --debug --verbose docker-toolbox
RemovePendingPackagesTask is now ready and waiting for PreRunMessage.
Sending message 'PreRunMessage' out if there are subscribers...
[Pending] Removing all pending packages that should not be considered installed...
The source 'https://chocolatey.org/api/v2/' evaluated to a 'normal' source type

NOTE: Hiding sensitive configuration data! Please double and triple
 check to be sure no sensitive data is shown, especially if copying
 output to a gist for review.
Configuration: CommandName='upgrade'|
CacheLocation='C:\Users\niha\AppData\Local\Temp\chocolatey'|
ContainsLegacyPackageInstalls='True'|
CommandExecutionTimeoutSeconds='2700'|WebRequestTimeoutSeconds='30'|
Sources='https://chocolatey.org/api/v2/'|SourceType='normal'|
Debug='True'|Verbose='True'|Trace='False'|Force='False'|Noop='False'|
HelpRequested='False'|UnsuccessfulParsing='False'|RegularOutput='True'|
QuietOutput='False'|PromptForConfirmation='False'|
AcceptLicense='False'|
AllowUnofficialBuild='False'|Input='docker-toolbox'|
AllVersions='False'|
SkipPackageInstallProvider='False'|PackageNames='docker-toolbox'|
Prerelease='False'|ForceX86='False'|OverrideArguments='False'|
NotSilent='False'|ApplyPackageParametersToDependencies='False'|
ApplyInstallArgumentsToDependencies='False'|IgnoreDependencies='False'|
AllowMultipleVersions='False'|AllowDowngrade='False'|
ForceDependencies='False'|Information.PlatformType='Windows'|
Information.PlatformVersion='10.0.15063.0'|
Information.PlatformName='Windows 10'|
Information.ChocolateyVersion='0.10.8.0'|
Information.ChocolateyProductVersion='0.10.8'|
Information.FullName='choco, Version=0.10.8.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb'|

Information.Is64BitOperatingSystem='True'|
Information.Is64BitProcess='True'|Information.IsInteractive='True'|
Information.UserName='niha'|Information.UserDomainName='EM'|
Information.IsUserAdministrator='True'|
Information.IsUserSystemAccount='False'|
Information.IsUserRemoteDesktop='False'|
Information.IsUserRemote='True'|
Information.IsProcessElevated='True'|
Information.IsLicensedVersion='False'|Information.LicenseType='Foss'|
Features.AutoUninstaller='True'|Features.ChecksumFiles='True'|
Features.AllowEmptyChecksums='False'|
Features.AllowEmptyChecksumsSecure='True'|
Features.FailOnAutoUninstaller='False'|
Features.FailOnStandardError='False'|Features.UsePowerShellHost='True'|
Features.LogEnvironmentValues='False'|Features.VirusCheck='False'|
Features.FailOnInvalidOrMissingLicense='False'|
Features.IgnoreInvalidOptionsSwitches='True'|
Features.UsePackageExitCodes='True'|
Features.UseFipsCompliantChecksums='False'|
Features.ShowNonElevatedWarnings='True'|
Features.ShowDownloadProgress='True'|
Features.StopOnFirstPackageFailure='False'|
Features.UseRememberedArgumentsForUpgrades='False'|
Features.ScriptsCheckLastExitCode='False'|
ListCommand.LocalOnly='False'|
ListCommand.IdOnly='False'|ListCommand.IncludeRegistryPrograms='False'|
ListCommand.PageSize='25'|ListCommand.Exact='False'|
ListCommand.ByIdOnly='False'|ListCommand.ByTagOnly='False'|
ListCommand.IdStartsWith='False'|ListCommand.OrderByPopularity='False'|
ListCommand.ApprovedOnly='False'|
ListCommand.DownloadCacheAvailable='False'|
ListCommand.NotBroken='False'|
ListCommand.IncludeVersionOverrides='False'|
UpgradeCommand.FailOnUnfound='False'|
UpgradeCommand.FailOnNotInstalled='False'|
UpgradeCommand.NotifyOnlyAvailableUpgrades='False'|
UpgradeCommand.ExcludePrerelease='False'|
NewCommand.AutomaticPackage='False'|
NewCommand.UseOriginalTemplate='False'|SourceCommand.Command='unknown'|
SourceCommand.Priority='0'|SourceCommand.BypassProxy='False'|
SourceCommand.AllowSelfService='False'|
SourceCommand.VisibleToAdminsOnly='False'|
FeatureCommand.Command='unknown'|ConfigCommand.Command='unknown'|
PinCommand.Command='unknown'|OutdatedCommand.IgnorePinned='False'|
Proxy.BypassOnLocal='True'|
_ Chocolatey:ChocolateyUpgradeCommand - Normal Run Mode _
Upgrading the following packages:
docker-toolbox
By upgrading you accept licenses for the packages.

You have docker-toolbox v17.06.0 installed. Version 17.06.2 is available based on your source(s).
Backing up existing docker-toolbox prior to operation.
Moving 'C:\ProgramData\chocolatey\lib\docker-toolbox'
 to 'C:\ProgramData\chocolatey\lib-bkp\docker-toolbox'
Attempting to create directory "C:\ProgramData\chocolatey\lib\docker-toolbox".
Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\docker-toolbox\docker-toolbox.nupkg"
 to "C:\ProgramData\chocolatey\lib\docker-toolbox\docker-toolbox.nupkg".
Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\docker-toolbox\docker-toolbox.nuspec"
 to "C:\ProgramData\chocolatey\lib\docker-toolbox\docker-toolbox.nuspec".
Attempting to create directory "C:\ProgramData\chocolatey\lib\docker-toolbox\tools".
Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\docker-toolbox\tools\chocolateyinstall.ps1"
 to "C:\ProgramData\chocolatey\lib\docker-toolbox\tools\chocolateyinstall.ps1".
Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\docker-toolbox\tools\chocolateyuninstall.ps1"
 to "C:\ProgramData\chocolatey\lib\docker-toolbox\tools\chocolateyuninstall.ps1".
Capturing package files in 'C:\ProgramData\chocolatey\lib\docker-toolbox'
 Found 'C:\ProgramData\chocolatey\lib\docker-toolbox\docker-toolbox.nupkg'
  with checksum '1859DC8C4DB71CF9E88DD6F6F8419F6A'
 Found 'C:\ProgramData\chocolatey\lib\docker-toolbox\docker-toolbox.nuspec'
  with checksum 'C8A96D91E4D882BE65578A4F386D8A49'
 Found 'C:\ProgramData\chocolatey\lib\docker-toolbox\tools\chocolateyinstall.ps1'
  with checksum '11642FBCAF629B88E14FF7CC3F6AB590'
 Found 'C:\ProgramData\chocolatey\lib\docker-toolbox\tools\chocolateyuninstall.ps1'
  with checksum 'F70BBC304B8D6A6CBCCA6EE7C340BB54'
[NuGet] Attempting to resolve dependency 'git.install (= 2.5.0)'.
docker-toolbox not upgraded. An error occurred during installation:
 Updating 'git.install 2.14.1' to 'git.install 2.6.0' failed. Unable to find a version of 'git' that is compatible with 'git.install 2.6.0'.
docker-toolbox package files upgrade completed. Performing other installation steps.
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.06.2\.arguments".
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.06.2\.extra".
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.06.2\.version".
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.06.2\.sxs".
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.06.2\.pin".
Attempting to delete directory "C:\ProgramData\chocolatey\lib-bad\docker-toolbox".
Sending message 'HandlePackageResultCompletedMessage' out if there are subscribers...
The upgrade of docker-toolbox was NOT successful.
docker-toolbox not upgraded. An error occurred during installation:
 Updating 'git.install 2.14.1' to 'git.install 2.6.0' failed. Unable to find a version of 'git' that is compatible with 'git.install 2.6.0'.
Moving 'C:\ProgramData\chocolatey\lib\docker-toolbox'
 to 'C:\ProgramData\chocolatey\lib-bad\docker-toolbox'
Attempting rollback
Moving 'C:\ProgramData\chocolatey\lib-bkp\docker-toolbox'
 to 'C:\ProgramData\chocolatey\lib\docker-toolbox'

Chocolatey upgraded 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - docker-toolbox (exited 1) - docker-toolbox not upgraded. An error occurred during installation:
 Updating 'git.install 2.14.1' to 'git.install 2.6.0' failed. Unable to find a version of 'git' that is compatible with 'git.install 2.6.0'.
Sending message 'PostRunMessage' out if there are subscribers...
Exiting with 1
```

**Comment by ferventcoder** [comment]
@NicoHaase that is definitely incorrect behavior. I think what you may be running into is multiple nupkgs confusing choco - take a look at https://chocolatey.org/docs/troubleshooting#already-referencing-a-newer-version-of-packagename and see if you have any stray nupkgs sitting around.

**Comment by NicoHaase** [comment]
Nope, that did not help. I used the script to remove these possibly broken files, but the update is still not possible. The verbose log of actions looks similar to the posted one

**Comment by ferventcoder** [comment]
Actually I wonder if you have another package that is restricting the version of git and Chocolatey it trying to correct that during upgrade.

**Comment by ferventcoder** [comment]
Output of `choco list -lo -r`?

**Comment by NicoHaase** [comment]
There you are:

> 7zip.install|16.4.0.20170506
7zip.portable|16.04
adobereader|2015.007.20033.02
adobereader-de|11.0.10
adobereader-update|17.012.20098
autohotkey|1.1.26.01
autohotkey.install|1.1.26.01
autohotkey.portable|1.1.26.01
chocolatey|0.10.8
chocolatey-core.extension|1.3.3
clink|0.4.8
dbgview|4.81
docker-toolbox|17.06.0
filezilla|3.28.0.20171005
Firefox|56.0.1
flashplayeractivex|27.0.0.170
flashplayerplugin|27.0.0.170
freemind|1.0.1
Ghostscript|9.22
Ghostscript.app|9.22
gimp|2.8.22.20171021
git|2.14.3
git.install|2.14.3
GoogleChrome|62.0.3202.62
hipchat|4.30.1.1663
InkScape|0.92.1
irfanview|4.50
irfanviewplugins|4.50
javaruntime|8.0.144
join.me|3.2.1.5233
jre8|8.0.144
libreoffice|5.4.2
mysql.workbench|6.3.8
notepadplusplus.install|7.5.1
openhardwaremonitor|0.8.0
owncloud-client|2.3.3.8250
pdfsam.install|3.3.2
pidgin|2.12.0
procexp|16.21.0.20170716
procmon|3.40
putty|0.70
putty.install|0.70
putty.portable|0.70
realtemp|3.70
sharex|11.9.1
sqlite.shell|3.10.1
sublime|2.0.2
sumatrapdf.install|3.1.2
superputty|1.4.0.8
superputty.install|1.4.0.8
teamviewer|12.0.83369
temp_BoxstarterPackage|1.0.0
tortoisegit|2.5.0.0
tortoisesvn|1.9.7.27907
totalcommander|9.10
ultravnc|1.2120
vagrant|2.0.0
vcredist2010|10.0.40219.2
vcredist2013|12.0.30501.20150616
virtualbox|5.1.26
vlc|2.2.6
wincdemu|4.1
windirstat|1.1.2.20161210
winmerge|2.14.0.20170224
WinPcap|4.1.3.20161116
winscp|5.11.2
winscp.install|5.11.2
wireshark|2.4.2

**Comment by ferventcoder** [comment]
@NicoHaase okay, I have something new for you to run:

~~~powershell
$nuspecs = gci $env:ChocolateyInstall\lib -recurse -filter *.nuspec

foreach ($nuspec in $nuspecs) {
  [xml]$nuspecContent = Get-Content $nuspec.FullName

  $dependencies = $nuspecContent.package.metadata.dependencies.dependency
  if ($dependencies -ne $null) {
    Write-Host "$($nuspecContent.package.metadata.id) v$($nuspecContent.package.metadata.version)"
    foreach ($dependency in $dependencies) {
      Write-Host " - $($dependency.id) - $($dependency.version)"
    }
  }
}
~~~

**Comment by ferventcoder** [comment]
updated code to run

**Comment by ferventcoder** [comment]
and another update to the code

**Comment by NicoHaase** [comment]
That's the result:

```
7zip.install v16.4.0.20170506 dependencies:
 - chocolatey-core.extension >= 1.2
adobereader-update v17.012.20098 dependencies:
 - adobereader  {Any Version}
autohotkey v1.1.26.01 dependencies:
 - autohotkey.install >= 1.1.26.01
autohotkey.install v1.1.26.01 dependencies:
 - chocolatey-core.extension >= 1.0
docker-toolbox v17.06.0 dependencies:
 - git.install >= 2.5.0
Firefox v56.0.1 dependencies:
 - chocolatey-core.extension >= 1.0
Ghostscript v9.22 dependencies:
 - ghostscript.app = [9.22]
Ghostscript.app v9.22 dependencies:
 - chocolatey-core.extension >= 1.2.0
git v2.14.3 dependencies:
 - git.install = [2.14.3]
git.install v2.14.3 dependencies:
 - chocolatey-core.extension >= 1.3.1
 - chocolatey >= 0.10.7
GoogleChrome v62.0.3202.62 dependencies:
 - chocolatey-core.extension >= 1.2.0
HipChat v4.30.1.1663 dependencies:
 - chocolatey-core.extension >= 1.0.4
InkScape v0.92.1 dependencies:
 - chocolatey-core.extension >= 1.0
IrfanView v4.50 dependencies:
 - autohotkey  {Any Version}
irfanviewplugins v4.50 dependencies:
 - irfanview >= 4.50
javaruntime v8.0.144 dependencies:
 - jre8 >= 8.0.144
mysql.workbench v6.3.8 dependencies:
 - vcredist2013  {Any Version}
notepadplusplus.install v7.5.1 dependencies:
 - chocolatey-core.extension >= 1.0.3
owncloud-client v2.3.3.8250 dependencies:
 - chocolatey-core.extension >= 1.0
pdfsam.install v3.3.2 dependencies:
 - javaruntime >= 8.0.0
putty v0.70 dependencies:
 - putty.portable = [0.70]
superputty v1.4.0.8 dependencies:
 - superputty.install = [1.4.0.8]
superputty.install v1.4.0.8 dependencies:
 - putty  {Any Version}
TotalCommander v9.10 dependencies:
 - chocolatey-core.extension >= 1.0.7
 - 7zip.portable >= 16.04
 - autohotkey.portable >= 1.1.24.05
 - chocolatey >= 0.10.5
virtualbox v5.1.26 dependencies:
 - chocolatey-core.extension >= 1.0.4
vlc v2.2.6 dependencies:
 - chocolatey-core.extension >= 1.0.4
WinPcap v4.1.3.20161116 dependencies:
 - autohotkey.portable  {Any Version}
 - chocolatey-core.extension >= 1.0
winscp v5.11.2 dependencies:
 - winscp.install = [5.11.2]
winscp.install v5.11.2 dependencies:
 - chocolatey-core.extension >= 1.3.1
```

**Comment by ferventcoder** [comment]
Another thought here - there may be a local cache of nupkgs that is not getting cleared. Are you familiar with those locations?

**Comment by ferventcoder** [comment]
The cache is supposed to be ignored, but let's ensure it is not causing the issue.

**Comment by NicoHaase** [comment]
How can I ensure that? Which folders should I clear or rename?

**Comment by ferventcoder** [comment]
In an explorer window, paste `%LocalAppData%\NuGet\Cache` and delete the nupkgs in there. You may also find a NuGet scratch folder in `%TEMP%` somewhere that should also be removed.

**Comment by NicoHaase** [comment]
I cleared the folder with cached nupkg-files and the whole %TEMP% folder, but the update still won't run :(

**Comment by ferventcoder** [comment]
Deeper checks now:

* Run with `-dv --trace` and capture the output (some won't go into logs)
* Turn on Fiddler and run, save that fiddler output somewhere

Provide both of those (you may want to scrub both first to ensure no sensitive data is exposed).

**Comment by NicoHaase** [comment]
That's it :)

<details>
<summary>Trace output</summary>

```Microsoft Windows [Version 10.0.15063]
(c) 2017 Microsoft Corporation. Alle Rechte vorbehalten.
Clink v0.4.8 [git:d565ad] Copyright (c) 2012-2016 Martin Ridgers
http://mridgers.github.io/clink


C:\WINDOWS\system32>cup -dv --trace docker-toolbox
Chocolatey v0.10.8
Chocolatey is running on Windows v 10.0.15063.0
Attempting to delete file "C:/ProgramData/chocolatey/choco.exe.old".
Attempting to delete file "C:\ProgramData\chocolatey\choco.exe.old".
Command line: "C:\ProgramData\chocolatey\choco.exe" upgrade -dv --trace docker-toolbox
Received arguments: upgrade -dv --trace docker-toolbox
RemovePendingPackagesTask is now ready and waiting for PreRunMessage.
Sending message 'PreRunMessage' out if there are subscribers...
[Pending] Removing all pending packages that should not be considered installed...
The source 'https://chocolatey.org/api/v2/' evaluated to a 'normal' source type

NOTE: Hiding sensitive configuration data! Please double and triple
 check to be sure no sensitive data is shown, especially if copying
 output to a gist for review.
Configuration: CommandName='upgrade'|
CacheLocation='C:\Users\niha\AppData\Local\Temp\chocolatey'|
ContainsLegacyPackageInstalls='True'|
CommandExecutionTimeoutSeconds='2700'|WebRequestTimeoutSeconds='30'|
Sources='https://chocolatey.org/api/v2/'|SourceType='normal'|
Debug='True'|Verbose='True'|Trace='True'|Force='False'|Noop='False'|
HelpRequested='False'|UnsuccessfulParsing='False'|RegularOutput='True'|
QuietOutput='False'|PromptForConfirmation='False'|
AcceptLicense='False'|
AllowUnofficialBuild='False'|Input='docker-toolbox'|
AllVersions='False'|
SkipPackageInstallProvider='False'|PackageNames='docker-toolbox'|
Prerelease='False'|ForceX86='False'|OverrideArguments='False'|
NotSilent='False'|ApplyPackageParametersToDependencies='False'|
ApplyInstallArgumentsToDependencies='False'|IgnoreDependencies='False'|
AllowMultipleVersions='False'|AllowDowngrade='False'|
ForceDependencies='False'|Information.PlatformType='Windows'|
Information.PlatformVersion='10.0.15063.0'|
Information.PlatformName='Windows 10'|
Information.ChocolateyVersion='0.10.8.0'|
Information.ChocolateyProductVersion='0.10.8'|
Information.FullName='choco, Version=0.10.8.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb'|

Information.Is64BitOperatingSystem='True'|
Information.Is64BitProcess='True'|Information.IsInteractive='True'|
Information.UserName='niha'|Information.UserDomainName='EM'|
Information.IsUserAdministrator='True'|
Information.IsUserSystemAccount='False'|
Information.IsUserRemoteDesktop='False'|
Information.IsUserRemote='True'|
Information.IsProcessElevated='True'|
Information.IsLicensedVersion='False'|Information.LicenseType='Foss'|
Features.AutoUninstaller='True'|Features.ChecksumFiles='True'|
Features.AllowEmptyChecksums='False'|
Features.AllowEmptyChecksumsSecure='True'|
Features.FailOnAutoUninstaller='False'|
Features.FailOnStandardError='False'|Features.UsePowerShellHost='True'|
Features.LogEnvironmentValues='False'|Features.VirusCheck='False'|
Features.FailOnInvalidOrMissingLicense='False'|
Features.IgnoreInvalidOptionsSwitches='True'|
Features.UsePackageExitCodes='True'|
Features.UseFipsCompliantChecksums='False'|
Features.ShowNonElevatedWarnings='True'|
Features.ShowDownloadProgress='True'|
Features.StopOnFirstPackageFailure='False'|
Features.UseRememberedArgumentsForUpgrades='False'|
Features.ScriptsCheckLastExitCode='False'|
ListCommand.LocalOnly='False'|
ListCommand.IdOnly='False'|ListCommand.IncludeRegistryPrograms='False'|
ListCommand.PageSize='25'|ListCommand.Exact='False'|
ListCommand.ByIdOnly='False'|ListCommand.ByTagOnly='False'|
ListCommand.IdStartsWith='False'|ListCommand.OrderByPopularity='False'|
ListCommand.ApprovedOnly='False'|
ListCommand.DownloadCacheAvailable='False'|
ListCommand.NotBroken='False'|
ListCommand.IncludeVersionOverrides='False'|
UpgradeCommand.FailOnUnfound='False'|
UpgradeCommand.FailOnNotInstalled='False'|
UpgradeCommand.NotifyOnlyAvailableUpgrades='False'|
UpgradeCommand.ExcludePrerelease='False'|
NewCommand.AutomaticPackage='False'|
NewCommand.UseOriginalTemplate='False'|SourceCommand.Command='unknown'|
SourceCommand.Priority='0'|SourceCommand.BypassProxy='False'|
SourceCommand.AllowSelfService='False'|
SourceCommand.VisibleToAdminsOnly='False'|
FeatureCommand.Command='unknown'|ConfigCommand.Command='unknown'|
PinCommand.Command='unknown'|OutdatedCommand.IgnorePinned='False'|
Proxy.BypassOnLocal='True'|
_ Chocolatey:ChocolateyUpgradeCommand - Normal Run Mode _
Upgrading the following packages:
docker-toolbox
By upgrading you accept licenses for the packages.
Initializing global mutex
Waiting on the mutext handle for 2000 milliseconds
Entered mutex to deserialize 'C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.06.0\.registry'
Initializing global mutex
Waiting on the mutext handle for 2000 milliseconds
Entered mutex to deserialize 'C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.06.0\.files'
System.Net Error: 0 : [7236] Die Proxyeinstellungen für URI "https://chocolatey.org/api/v2/" können nicht abgerufen werden. Fehlercode: 12180.
System.Net Information: 0 : [7236] Associating HttpWebRequest#4032828 with ServicePoint#37489757
System.Net Information: 0 : [7236] Associating Connection#64828693 with HttpWebRequest#4032828
System.Net.Sockets Information: 0 : [7236] Socket#10104599 - Die Verbindung von 10.13.110.186:49824 mit 104.20.74.28:443 wurde hergestellt.
System.Net Information: 0 : [7236] Connection#64828693 - Die Verbindung von 10.13.110.186:49824 mit 104.20.74.28:443 wurde hergestellt.
System.Net Information: 0 : [7236] TlsStream#7141266::.ctor(host=chocolatey.org, #certs=0)
System.Net Information: 0 : [7236] Associating HttpWebRequest#4032828 with ConnectStream#44313942
System.Net Information: 0 : [7236] HttpWebRequest#4032828 - Request: GET /api/v2/ HTTP/1.0

System.Net Information: 0 : [7236] ConnectStream#44313942 - Header
{
User-Agent: NuGet/0.10.8.0 (Microsoft Windows NT 10.0.15063.0)
Host: chocolatey.org
} werden gesendet.
System.Net Information: 0 : [7236] SecureChannel#34106743::.ctor(hostname=chocolatey.org, #clientCertificates=0, encryptionPolicy=RequireEncryption)
System.Net Information: 0 : [7236] Sicherheitspakete werden enumeriert:
System.Net Information: 0 : [7236]     Negotiate
System.Net Information: 0 : [7236]     NegoExtender
System.Net Information: 0 : [7236]     Kerberos
System.Net Information: 0 : [7236]     NTLM
System.Net Information: 0 : [7236]     TSSSP
System.Net Information: 0 : [7236]     pku2u
System.Net Information: 0 : [7236]     CloudAP
System.Net Information: 0 : [7236]     WDigest
System.Net Information: 0 : [7236]     Schannel
System.Net Information: 0 : [7236]     Microsoft Unified Security Protocol Provider
System.Net Information: 0 : [7236]     Default TLS SSP
System.Net Information: 0 : [7236]     CREDSSP
System.Net Information: 0 : [7236] SecureChannel#34106743 - Noch 0 Clientzertifikate zum Auswählen verfügbar.
System.Net Information: 0 : [7236] AcquireCredentialsHandle(package = Microsoft Unified Security Protocol Provider, intent  = Outbound, scc     = System.Net.SecureCredential)
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = (null), targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Länge von In-Buffer = 0, Länge von Out-Buffer = 174, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 93, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = OK).
System.Net Information: 0 : [7236] Remotezertifikat: [Version]
  V3

[Subject]
  CN=*.chocolatey.org, OU=Domain Control Validated
  Simple Name: *.chocolatey.org
  DNS Name: chocolatey.org

[Issuer]
  CN=Go Daddy Secure Certificate Authority - G2, OU=http://certs.godaddy.com/repository/, O="GoDaddy.com, Inc.", L=Scottsdale, S=Arizona, C=US
  Simple Name: Go Daddy Secure Certificate Authority - G2
  DNS Name: Go Daddy Secure Certificate Authority - G2

[Serial Number]
  518FA4063FE8C5D5

[Not Before]
  03.05.2017 19:25:01

[Not After]
  03.05.2019 19:25:01

[Thumbprint]
  B643AFB4DF5FF4F88B8C662DD55F496230F90747

[Signature Algorithm]
  sha256RSA(1.2.840.113549.1.1.11)

[Public Key]
  Algorithm: RSA
  Length: 2048
  Key Blob: 30 82 01 0a 02 82 01 01 00 8b 35 28 4d 35 56 9c 01 de 92 9a 89 b2 f0 d3 8b ae af 22 c5 a4 1c 05 25 6d e4 7f 49 b2 5e df 80 23 b5 e2 c8 58 dc 02 f0 30 80 8e e1 60 29 c5 08 ec b0 57 6d 44 a9 38 1e fa 45 6e 6d 37 40 fd 37 89 ce a3 07 41 aa f8 1e 54 11 eb b5 84 da 39 53 8d 2a a1 99 a4 b2 35 5d 82 35 a9 b4 06 19....
System.Net Information: 0 : [7236] SecureChannel#34106743 - Das Remotezertifikat wurde vom Benutzer als gültig bestätigt.
System.Net Information: 0 : [7236] ProcessAuthentication(Protokoll=Tls12, Verschlüsselungsverfahren=Aes128 128 Bitstärke, Hash=Sha256 0 Bitstärke, Schlüsselaustausch=44550 255 Bitstärke).
System.Net Information: 0 : [7236] Connection#64828693 - Empfangene Statusleiste: Version = 1.1, StatusCode = 200, StatusDescription = OK.
System.Net Information: 0 : [7236] Connection#64828693 - Header
{
Connection: close
DataServiceVersion: 1.0;
CF-Cache-Status: HIT
Strict-Transport-Security: max-age=12960000
X-Content-Type-Options: nosniff
CF-RAY: 3b973f2b9f656355-FRA
Content-Length: 397
Cache-Control: public, max-age=31536000
Content-Type: application/xml;charset=utf-8
Date: Mon, 06 Nov 2017 10:07:36 GMT
Expires: Tue, 06 Nov 2018 10:07:36 GMT
Set-Cookie: __cfduid=db7288084f1ed8b04b12f70d6c5a82e531509962856; expires=Tue, 06-Nov-18 10:07:36 GMT; path=/; domain=.chocolatey.org; HttpOnly
Server: cloudflare-nginx
} wurden empfangen.
System.Net Information: 0 : [7236] ConnectStream#47362231::ConnectStream(Es wurden 397 Bytes gepuffert.)
System.Net Information: 0 : [7236] Associating HttpWebRequest#4032828 with ConnectStream#47362231
System.Net Information: 0 : [7236] Associating HttpWebRequest#4032828 with HttpWebResponse#43205102
System.Net Information: 0 : [7236] Associating HttpWebRequest#6385742 with ServicePoint#37489757
System.Net Information: 0 : [7236] Associating Connection#25425822 with HttpWebRequest#6385742
System.Net.Sockets Information: 0 : [7236] Socket#31665793 - Die Verbindung von 10.13.110.186:49825 mit 104.20.74.28:443 wurde hergestellt.
System.Net Information: 0 : [7236] Connection#25425822 - Die Verbindung von 10.13.110.186:49825 mit 104.20.74.28:443 wurde hergestellt.
System.Net Information: 0 : [7236] TlsStream#61115195::.ctor(host=chocolatey.org, #certs=0)
System.Net Information: 0 : [7236] Associating HttpWebRequest#6385742 with ConnectStream#51484875
System.Net Information: 0 : [7236] HttpWebRequest#6385742 - Request: GET /api/v2/$metadata HTTP/1.0

System.Net Information: 0 : [7236] ConnectStream#51484875 - Header
{
User-Agent: NuGet/0.10.8.0 (Microsoft Windows NT 10.0.15063.0)
Host: chocolatey.org
} werden gesendet.
System.Net Information: 0 : [7236] SecureChannel#12053474::.ctor(hostname=chocolatey.org, #clientCertificates=0, encryptionPolicy=RequireEncryption)
System.Net Information: 0 : [7236] SecureChannel#12053474 - Noch 0 Clientzertifikate zum Auswählen verfügbar.
System.Net Information: 0 : [7236] Das zwischengespeicherte Anmeldeinformationshandle wird verwendet.
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = (null), targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Länge von In-Buffer = 0, Länge von Out-Buffer = 382, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 51, zurückgegebener Code = OK).
System.Net Information: 0 : [7236] Remotezertifikat: [Version]
  V3

[Subject]
  CN=*.chocolatey.org, OU=Domain Control Validated
  Simple Name: *.chocolatey.org
  DNS Name: chocolatey.org

[Issuer]
  CN=Go Daddy Secure Certificate Authority - G2, OU=http://certs.godaddy.com/repository/, O="GoDaddy.com, Inc.", L=Scottsdale, S=Arizona, C=US
  Simple Name: Go Daddy Secure Certificate Authority - G2
  DNS Name: Go Daddy Secure Certificate Authority - G2

[Serial Number]
  518FA4063FE8C5D5

[Not Before]
  03.05.2017 19:25:01

[Not After]
  03.05.2019 19:25:01

[Thumbprint]
  B643AFB4DF5FF4F88B8C662DD55F496230F90747

[Signature Algorithm]
  sha256RSA(1.2.840.113549.1.1.11)

[Public Key]
  Algorithm: RSA
  Length: 2048
  Key Blob: 30 82 01 0a 02 82 01 01 00 8b 35 28 4d 35 56 9c 01 de 92 9a 89 b2 f0 d3 8b ae af 22 c5 a4 1c 05 25 6d e4 7f 49 b2 5e df 80 23 b5 e2 c8 58 dc 02 f0 30 80 8e e1 60 29 c5 08 ec b0 57 6d 44 a9 38 1e fa 45 6e 6d 37 40 fd 37 89 ce a3 07 41 aa f8 1e 54 11 eb b5 84 da 39 53 8d 2a a1 99 a4 b2 35 5d 82 35 a9 b4 06 19....
System.Net Information: 0 : [7236] SecureChannel#12053474 - Das Remotezertifikat wurde vom Benutzer als gültig bestätigt.
System.Net Information: 0 : [7236] ProcessAuthentication(Protokoll=Tls12, Verschlüsselungsverfahren=Aes128 128 Bitstärke, Hash=Sha256 0 Bitstärke, Schlüsselaustausch=44550 255 Bitstärke).
System.Net Information: 0 : [7236] Connection#25425822 - Empfangene Statusleiste: Version = 1.1, StatusCode = 200, StatusDescription = OK.
System.Net Information: 0 : [7236] Connection#25425822 - Header
{
Connection: close
Vary: Accept-Encoding
DataServiceVersion: 2.0;
CF-Cache-Status: HIT
Strict-Transport-Security: max-age=12960000
X-Content-Type-Options: nosniff
CF-RAY: 3b973f2c1975650b-FRA
Cache-Control: public, max-age=86400
Content-Type: application/xml;charset=utf-8
Date: Mon, 06 Nov 2017 10:07:36 GMT
Expires: Tue, 07 Nov 2017 10:07:36 GMT
Set-Cookie: __cfduid=d8dc81b61329d1806732f13b02ec5e89d1509962856; expires=Tue, 06-Nov-18 10:07:36 GMT; path=/; domain=.chocolatey.org; HttpOnly
Server: cloudflare-nginx
} wurden empfangen.
System.Net Information: 0 : [7236] ConnectStream#32901400::ConnectStream(Es wurden -1 Bytes gepuffert.)
System.Net Information: 0 : [7236] Associating HttpWebRequest#6385742 with ConnectStream#32901400
System.Net Information: 0 : [7236] Associating HttpWebRequest#6385742 with HttpWebResponse#17228638
System.Net Information: 0 : [7236] ContentLength=-1
System.Net Information: 0 : [7236] Associating HttpWebRequest#38226137 with ServicePoint#37489757
System.Net Information: 0 : [7236] Associating Connection#16129352 with HttpWebRequest#38226137
System.Net.Sockets Information: 0 : [7236] Socket#580617 - Die Verbindung von 10.13.110.186:49826 mit 104.20.74.28:443 wurde hergestellt.
System.Net Information: 0 : [7236] Connection#16129352 - Die Verbindung von 10.13.110.186:49826 mit 104.20.74.28:443 wurde hergestellt.
System.Net Information: 0 : [7236] TlsStream#27341666::.ctor(host=chocolatey.org, #certs=0)
System.Net Information: 0 : [7236] Associating HttpWebRequest#38226137 with ConnectStream#12453028
System.Net Information: 0 : [7236] HttpWebRequest#38226137 - Request: GET /api/v2/FindPackagesById()?id='docker-toolbox' HTTP/1.1

System.Net Information: 0 : [7236] ConnectStream#12453028 - Header
{
DataServiceVersion: 1.0;NetFx
MaxDataServiceVersion: 2.0;NetFx
User-Agent: Chocolatey Core
Accept: application/atom+xml,application/xml
Accept-Charset: UTF-8
Host: chocolatey.org
Accept-Encoding: gzip, deflate
} werden gesendet.
System.Net Information: 0 : [7236] SecureChannel#42890253::.ctor(hostname=chocolatey.org, #clientCertificates=0, encryptionPolicy=RequireEncryption)
System.Net Information: 0 : [7236] SecureChannel#42890253 - Noch 0 Clientzertifikate zum Auswählen verfügbar.
System.Net Information: 0 : [7236] Das zwischengespeicherte Anmeldeinformationshandle wird verwendet.
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = (null), targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Länge von In-Buffer = 0, Länge von Out-Buffer = 382, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 0, zurückgegebener Code = ContinueNeeded).
System.Net Information: 0 : [7236] InitializeSecurityContext(credential = System.Net.SafeFreeCredential_SECURITY, context = 1c3b0a80:245b47c4790, targetName = chocolatey.org, inFlags = ReplayDetect, SequenceDetect, Confidentiality, AllocateMemory, InitManualCredValidation)
System.Net Information: 0 : [7236] InitializeSecurityContext(Anzahl von In-Buffers = 2, Länge von Out-Buffer = 51, zurückgegebener Code = OK).
System.Net Information: 0 : [7236] Remotezertifikat: [Version]
  V3

[Subject]
  CN=*.chocolatey.org, OU=Domain Control Validated
  Simple Name: *.chocolatey.org
  DNS Name: chocolatey.org

[Issuer]
  CN=Go Daddy Secure Certificate Authority - G2, OU=http://certs.godaddy.com/repository/, O="GoDaddy.com, Inc.", L=Scottsdale, S=Arizona, C=US
  Simple Name: Go Daddy Secure Certificate Authority - G2
  DNS Name: Go Daddy Secure Certificate Authority - G2

[Serial Number]
  518FA4063FE8C5D5

[Not Before]
  03.05.2017 19:25:01

[Not After]
  03.05.2019 19:25:01

[Thumbprint]
  B643AFB4DF5FF4F88B8C662DD55F496230F90747

[Signature Algorithm]
  sha256RSA(1.2.840.113549.1.1.11)

[Public Key]
  Algorithm: RSA
  Length: 2048
  Key Blob: 30 82 01 0a 02 82 01 01 00 8b 35 28 4d 35 56 9c 01 de 92 9a 89 b2 f0 d3 8b ae af 22 c5 a4 1c 05 25 6d e4 7f 49 b2 5e df 80 23 b5 e2 c8 58 dc 02 f0 30 80 8e e1 60 29 c5 08 ec b0 57 6d 44 a9 38 1e fa 45 6e 6d 37 40 fd 37 89 ce a3 07 41 aa f8 1e 54 11 eb b5 84 da 39 53 8d 2a a1 99 a4 b2 35 5d 82 35 a9 b4 06 19....
System.Net Information: 0 : [7236] SecureChannel#42890253 - Das Remotezertifikat wurde vom Benutzer als gültig bestätigt.
System.Net Information: 0 : [7236] ProcessAuthentication(Protokoll=Tls12, Verschlüsselungsverfahren=Aes128 128 Bitstärke, Hash=Sha256 0 Bitstärke, Schlüsselaustausch=44550 255 Bitstärke).
System.Net Information: 0 : [7236] Connection#16129352 - Empfangene Statusleiste: Version = 1.1, StatusCode = 200, StatusDescription = OK.
System.Net Information: 0 : [7236] Connection#16129352 - Header
{
Connection: keep-alive
DataServiceVersion: 2.0;
CF-Cache-Status: EXPIRED
Vary: Accept-Encoding
Strict-Transport-Security: max-age=12960000
X-Content-Type-Options: nosniff
CF-RAY: 3b973f2cd8106355-FRA
Content-Length: 37035
Cache-Control: public, max-age=1800
Content-Type: application/atom+xml;charset=utf-8
Date: Mon, 06 Nov 2017 10:07:36 GMT
Expires: Mon, 06 Nov 2017 10:37:36 GMT
Set-Cookie: __cfduid=da084aea8ae181a3e7e673b5354fabe781509962856; expires=Tue, 06-Nov-18 10:07:36 GMT; path=/; domain=.chocolatey.org; HttpOnly
Server: cloudflare-nginx
} wurden empfangen.
System.Net Information: 0 : [7236] ConnectStream#65623374::ConnectStream(Es wurden 37035 Bytes gepuffert.)
System.Net Information: 0 : [7236] Associating HttpWebRequest#38226137 with ConnectStream#65623374
System.Net Information: 0 : [7236] Associating HttpWebRequest#38226137 with HttpWebResponse#29971633
System.Net Information: 0 : [7236] ContentLength=37035

You have docker-toolbox v17.06.0 installed. Version 17.10.0 is available based on your source(s).
Backing up existing docker-toolbox prior to operation.
Moving 'C:\ProgramData\chocolatey\lib\docker-toolbox'
 to 'C:\ProgramData\chocolatey\lib-bkp\docker-toolbox'
Attempting to create directory "C:\ProgramData\chocolatey\lib\docker-toolbox".
Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\docker-toolbox\docker-toolbox.nupkg"
 to "C:\ProgramData\chocolatey\lib\docker-toolbox\docker-toolbox.nupkg".
Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\docker-toolbox\docker-toolbox.nuspec"
 to "C:\ProgramData\chocolatey\lib\docker-toolbox\docker-toolbox.nuspec".
Attempting to create directory "C:\ProgramData\chocolatey\lib\docker-toolbox\tools".
Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\docker-toolbox\tools\chocolateyinstall.ps1"
 to "C:\ProgramData\chocolatey\lib\docker-toolbox\tools\chocolateyinstall.ps1".
Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\docker-toolbox\tools\chocolateyuninstall.ps1"
 to "C:\ProgramData\chocolatey\lib\docker-toolbox\tools\chocolateyuninstall.ps1".
Capturing package files in 'C:\ProgramData\chocolatey\lib\docker-toolbox'
 Found 'C:\ProgramData\chocolatey\lib\docker-toolbox\docker-toolbox.nupkg'
  with checksum '1859DC8C4DB71CF9E88DD6F6F8419F6A'
 Found 'C:\ProgramData\chocolatey\lib\docker-toolbox\docker-toolbox.nuspec'
  with checksum 'C8A96D91E4D882BE65578A4F386D8A49'
 Found 'C:\ProgramData\chocolatey\lib\docker-toolbox\tools\chocolateyinstall.ps1'
  with checksum '11642FBCAF629B88E14FF7CC3F6AB590'
 Found 'C:\ProgramData\chocolatey\lib\docker-toolbox\tools\chocolateyuninstall.ps1'
  with checksum 'F70BBC304B8D6A6CBCCA6EE7C340BB54'
[NuGet] Attempting to resolve dependency 'git.install (= 2.5.0)'.
System.Net Information: 0 : [7236] Associating HttpWebRequest#19315616 with ServicePoint#37489757
System.Net Information: 0 : [7236] Associating Connection#16129352 with HttpWebRequest#19315616
System.Net Information: 0 : [7236] Associating HttpWebRequest#19315616 with ConnectStream#13128364
System.Net Information: 0 : [7236] HttpWebRequest#19315616 - Request: GET /api/v2/Packages()?$filter=tolower(Id)%20eq%20'git'&$orderby=Id&$skip=0&$top=30 HTTP/1.1

System.Net Information: 0 : [7236] ConnectStream#13128364 - Header
{
DataServiceVersion: 1.0;NetFx
MaxDataServiceVersion: 2.0;NetFx
User-Agent: Chocolatey Core
Accept: application/atom+xml,application/xml
Accept-Charset: UTF-8
Host: chocolatey.org
Accept-Encoding: gzip, deflate
} werden gesendet.
System.Net Information: 0 : [7236] Connection#16129352 - Empfangene Statusleiste: Version = 1.1, StatusCode = 200, StatusDescription = OK.
System.Net Information: 0 : [7236] Connection#16129352 - Header
{
Connection: keep-alive
DataServiceVersion: 2.0;
CF-Cache-Status: EXPIRED
Vary: Accept-Encoding
Strict-Transport-Security: max-age=12960000
X-Content-Type-Options: nosniff
CF-RAY: 3b973f4aaa906355-FRA
Content-Length: 168251
Cache-Control: public, max-age=10800
Content-Type: application/atom+xml;charset=utf-8
Date: Mon, 06 Nov 2017 10:07:41 GMT
Expires: Mon, 06 Nov 2017 13:07:41 GMT
Set-Cookie: __cfduid=d57cfdac276b73f1a6f72ed02aa7665351509962861; expires=Tue, 06-Nov-18 10:07:41 GMT; path=/; domain=.chocolatey.org; HttpOnly
Server: cloudflare-nginx
} wurden empfangen.
System.Net Information: 0 : [7236] ConnectStream#59773647::ConnectStream(Es wurden 168251 Bytes gepuffert.)
System.Net Information: 0 : [7236] Associating HttpWebRequest#19315616 with ConnectStream#59773647
System.Net Information: 0 : [7236] Associating HttpWebRequest#19315616 with HttpWebResponse#17946170
System.Net Information: 0 : [7236] ContentLength=168251
System.Net Information: 0 : [7236] Associating HttpWebRequest#36296961 with ServicePoint#37489757
System.Net Information: 0 : [7236] Associating Connection#16129352 with HttpWebRequest#36296961
System.Net Information: 0 : [7236] Associating HttpWebRequest#36296961 with ConnectStream#35008814
System.Net Information: 0 : [7236] HttpWebRequest#36296961 - Request: GET /api/v2/Packages()?$filter=tolower(Id)%20eq%20'git'&$orderby=Id&$skip=30&$top=30 HTTP/1.1

System.Net Information: 0 : [7236] ConnectStream#35008814 - Header
{
DataServiceVersion: 1.0;NetFx
MaxDataServiceVersion: 2.0;NetFx
User-Agent: Chocolatey Core
Accept: application/atom+xml,application/xml
Accept-Charset: UTF-8
Host: chocolatey.org
Accept-Encoding: gzip, deflate
} werden gesendet.
System.Net Information: 0 : [7236] Connection#16129352 - Empfangene Statusleiste: Version = 1.1, StatusCode = 200, StatusDescription = OK.
System.Net Information: 0 : [7236] Connection#16129352 - Header
{
Connection: keep-alive
DataServiceVersion: 2.0;
CF-Cache-Status: EXPIRED
Vary: Accept-Encoding
Strict-Transport-Security: max-age=12960000
X-Content-Type-Options: nosniff
CF-RAY: 3b973f4fdd586355-FRA
Content-Length: 178201
Cache-Control: public, max-age=10800
Content-Type: application/atom+xml;charset=utf-8
Date: Mon, 06 Nov 2017 10:07:42 GMT
Expires: Mon, 06 Nov 2017 13:07:42 GMT
Set-Cookie: __cfduid=d10a2d18bd71c68048cfa9e59c2a656ec1509962862; expires=Tue, 06-Nov-18 10:07:42 GMT; path=/; domain=.chocolatey.org; HttpOnly
Server: cloudflare-nginx
} wurden empfangen.
System.Net Information: 0 : [7236] ConnectStream#2805130::ConnectStream(Es wurden 178201 Bytes gepuffert.)
System.Net Information: 0 : [7236] Associating HttpWebRequest#36296961 with ConnectStream#2805130
System.Net Information: 0 : [7236] Associating HttpWebRequest#36296961 with HttpWebResponse#3019398
System.Net Information: 0 : [7236] ContentLength=178201
System.Net Information: 0 : [7952] Associating HttpWebRequest#36408330 with ServicePoint#37489757
System.Net Information: 0 : [7952] Associating Connection#16129352 with HttpWebRequest#36408330
System.Net Information: 0 : [7952] Associating HttpWebRequest#36408330 with ConnectStream#4930960
System.Net Information: 0 : [7952] HttpWebRequest#36408330 - Request: GET /api/v2/Packages()?$filter=tolower(Id)%20eq%20'git'&$orderby=Id&$skip=60&$top=30 HTTP/1.1

System.Net Information: 0 : [7952] ConnectStream#4930960 - Header
{
DataServiceVersion: 1.0;NetFx
MaxDataServiceVersion: 2.0;NetFx
User-Agent: Chocolatey Core
Accept: application/atom+xml,application/xml
Accept-Charset: UTF-8
Host: chocolatey.org
Accept-Encoding: gzip, deflate
} werden gesendet.
System.Net Information: 0 : [7952] Connection#16129352 - Empfangene Statusleiste: Version = 1.1, StatusCode = 200, StatusDescription = OK.
System.Net Information: 0 : [7952] Connection#16129352 - Header
{
Connection: keep-alive
DataServiceVersion: 2.0;
CF-Cache-Status: EXPIRED
Vary: Accept-Encoding
Strict-Transport-Security: max-age=12960000
X-Content-Type-Options: nosniff
CF-RAY: 3b973f54a84e6355-FRA
Content-Length: 11925
Cache-Control: public, max-age=10800
Content-Type: application/atom+xml;charset=utf-8
Date: Mon, 06 Nov 2017 10:07:42 GMT
Expires: Mon, 06 Nov 2017 13:07:42 GMT
Set-Cookie: __cfduid=d10a2d18bd71c68048cfa9e59c2a656ec1509962862; expires=Tue, 06-Nov-18 10:07:42 GMT; path=/; domain=.chocolatey.org; HttpOnly
Server: cloudflare-nginx
} wurden empfangen.
System.Net Information: 0 : [7952] ConnectStream#48227816::ConnectStream(Es wurden 11925 Bytes gepuffert.)
System.Net Information: 0 : [7952] Associating HttpWebRequest#36408330 with ConnectStream#48227816
System.Net Information: 0 : [7952] Associating HttpWebRequest#36408330 with HttpWebResponse#39598739
System.Net Information: 0 : [7952] ContentLength=11925
docker-toolbox not upgraded. An error occurred during installation:
 Updating 'git.install 2.15.0' to 'git.install 2.6.0' failed. Unable to find a version of 'git' that is compatible with 'git.install 2.6.0'.
docker-toolbox package files upgrade completed. Performing other installation steps.
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.10.0\.arguments".
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.10.0\.extra".
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.10.0\.version".
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.10.0\.sxs".
Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\docker-toolbox.17.10.0\.pin".
Attempting to delete directory "C:\ProgramData\chocolatey\lib-bad\docker-toolbox".
Sending message 'HandlePackageResultCompletedMessage' out if there are subscribers...
The upgrade of docker-toolbox was NOT successful.
docker-toolbox not upgraded. An error occurred during installation:
 Updating 'git.install 2.15.0' to 'git.install 2.6.0' failed. Unable to find a version of 'git' that is compatible with 'git.install 2.6.0'.
Moving 'C:\ProgramData\chocolatey\lib\docker-toolbox'
 to 'C:\ProgramData\chocolatey\lib-bad\docker-toolbox'
Attempting rollback
Moving 'C:\ProgramData\chocolatey\lib-bkp\docker-toolbox'
 to 'C:\ProgramData\chocolatey\lib\docker-toolbox'

Chocolatey upgraded 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - docker-toolbox (exited 1) - docker-toolbox not upgraded. An error occurred during installation:
 Updating 'git.install 2.15.0' to 'git.install 2.6.0' failed. Unable to find a version of 'git' that is compatible with 'git.install 2.6.0'.
Sending message 'PostRunMessage' out if there are subscribers...
Exiting with 1

C:\WINDOWS\system32>
```
</details>

What I find interesting is that the script outputting dependencies writes that docker-toolbox expects git.install >= 2.5.0 (so any version after that), while the installer wants to install version 2.5.0 and explicitly no other.

How to work with Fiddler? I don't know that tool

**Comment by NicoHaase** [comment]
@ferventcoder do you have any more hints about what could have gone wrong?

**Comment by NicoHaase** [comment]
@ferventcoder any news on this? The package was updated to 18.01.0, but updating still fails

**Comment by ferventcoder** [comment]
@NicoHaase fiddler is at https://www.telerik.com/fiddler.

**Comment by brimwats** [comment]
This is an error in several places I believe. Look at https://github.com/chocolatey/choco/issues/227

**Comment by corbob** [comment]
I have tried and tried and tried to reproduce this, and the closest I can get is this error occurring with `chocolatey` version `0.10.8` installed and running the command `choco upgrade docker-toolbox --version 16.06.0 --allow-downgrade`. All other scenarios even with all packages loaded into a private repository result in it behaving exactly as I'd expect it to and not trying at all to downgrade anything.

At this point I'll ask if anyone is currently able to reproduce this behaviour, and if they're able to provide steps to reproduce it on a fresh system (or in Windows Sandbox).

**Comment by NicoHaase** [comment]
@corbob this bug report is nearly six years old, and I'm no longer using Chocolatey, so I can't tell you whether this is resolved or not

**Comment by corbob** [comment]
Thank you for the update @NicoHaase. I'll close off this issue. Should it happen again we can always re-open or open a new issue.
